### ISSUE TITLE
Namespace not specified. Specify a namespace in the module's build file

### ISSUE THREAD

**Issue by dharambudh1** [PhonePe/phonepe-pg-sdk-flutter#19]
I am facing an error after specifying `phonepe_payment_sdk` in the `pubspec.yaml` file.

![Pasted image (2)](https://github.com/user-attachments/assets/1fc98c2e-7e46-4f8d-a7a1-f706916859dc)

Download logs:  [error.txt](https://github.com/user-attachments/files/16358521/error.txt)

Here are my PhonePe SDK version (from: https://pub.dev/packages/phonepe_payment_sdk):
```
phonepe_payment_sdk: ^2.0.2
```

Here are my Flutter Doctor Summary:
```
[✓] Flutter (Channel stable, 3.19.3, on Ubuntu 24.04 LTS 6.8.0-38-generic, locale en_US.UTF-8)
[✓] Android toolchain - develop for Android devices (Android SDK version 35.0.0)
[✓] Android Studio (version 2024.1)
```

Here are my SDK & Gradle configurations. (It complies with the latest Google Play Store policies).

build.gradle:
```
minSdkVersion 24
targetSdk 35
compileSdk 35
```

settings.gradle:
```
id "com.android.application" version "8.5.0" apply false
id 'com.google.gms.google-services' version '4.4.1' apply false
id "org.jetbrains.kotlin.android" version "1.9.23" apply false
```

gradle-wrapper.properties:
```
distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-all.zip
```

**Comment by dharambudh1** [comment]
I have addressed the issue by including `namespace 'com.phonepe.phonepe_payment_sdk'` in the `build.gradle` file at `/home/dharam/.pub-cache/hosted/pub.dev/phonepe_payment_sdk-2.0.2/android/build.gradle`.

Could you please confirm if this is the appropriate solution? Additionally, I would like to know if there is a way for me to contribute to this project.

![Pasted image](https://github.com/user-attachments/assets/2a57a9ad-d2cc-46f9-aa15-ba6291ae9162)

**Comment by dharambudh1** [comment]
@rajanarora27, @NitinVermaPpe, @tejab41097, we can consider this issue resolved as it has been addressed in `phonepe_payment_sdk` version 2.0.3. You can review the details of the update [here](https://pub.dev/packages/phonepe_payment_sdk/versions/2.0.3).

Thank you for your attention to this matter.
